HadesArchitect/GrafanaCassandraDatasource
# Variable query support

Closes #168 

To test: 
- Make sure that C* is created new so sample-data.sh script will create new table
- Add query variable to a dashboard
<img width="767" alt="image" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/1742301/247f661b-3083-4a05-ab05-88ae4783f4a8">
- Check both fetching variable values using single column and two columns (label is present or not) - results will appear on the bottom 
- Create a new visualisation using that variable
<img width="905" alt="image" src="https://github.com/HadesArchitect/GrafanaCassandraDatasource/assets/1742301/25dcc9e5-1fca-4076-bf97-4a7422d0ae32">
- Play with variables
